src/providers/runProvider.ts

```diff
@@ -213,17 +211,17 @@ export class RunTreeDataProvider implements vscode.TreeDataProvider<RunTreeItem>
         const included = data.included ?? ([] as (Plan | Apply | User)[]);
 
         included.forEach((item) => {
-            if (item.type === 'plans') {
+            if ((item as Plan).type === 'plans') {
                 const plan = item as Plan;
-                if (plan.attributes?.status !== 'unreachable') {
+                if (plan.attributes?.status !== 'unreachable' && plan.attributes?.status !== 'pending') {
                     plans.set(item.id as string, plan);
                 }
-            } else if (item.type === 'applies') {
+            } else if ((item as Apply).type === 'applies') {
                 const apply = item as Apply;
-                if (apply.attributes?.status !== 'unreachable') {
+                if (apply.attributes?.status !== 'unreachable' && apply.attributes?.status !== 'pending') {
                     applies.set(item.id as string, apply);
                 }
-            } else if (item.type === 'users') {
+            } else if ((item as User).type === 'users') {
```

Comment:
@penja Instead of type casting you can use[ type predicates](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) here.
For example:
```ts
function isPlan(pet: unknown): pet is Plan {
    return !!(pet && typeof pet === 'object' && 'type' in pet && pet?.type === 'plans');
}

included.forEach((item) => {
if (isPlan(item)) {
    const plan = item;
    if (plan.attributes?.status !== 'unreachable' && plan.attributes?.status !== 'pending') {
        plans.set(String(item.id), plan);
    }
}
```

Reply:
@111crb111 👍🏻 will fix on the next release.